.eslintrc.js

```diff
@@ -11,7 +11,10 @@ module.exports = {
 		es6: true
 	},
 	parserOptions: {
-		ecmaVersion: 2017
+		ecmaVersion: 2017,
+		ecmaFeatures: {
+			experimentalObjectRestSpread: true
```

Comment:
This would allow `...` which is not support in node 6. We don't want to allow that.

Reply:
yes, that change is I fixed this line error, maybe can use another way
https://github.com/webpack/webpack/blob/9b02c19c8a34df7022052453ec91e1429f1f6d97/.eslintrc.js#L58